backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -73,8 +73,9 @@ def process_row_inner(self, sample_container, measures, process, process_measure
                 self.errors['library_volume'] = f'The library\'s computed final volume would be less than zero ({final_volume}). Please verify the volume currently stored for the library.'
 
         # library size
+        # TODO: I modified to optional if sample is pool. Is the assumption correct?
```

Comment:
yes as far as i know, lib_qc on a pool will not be setting lib_size since they should have all been QCed before being pooled. The QC will update the concentration as nM and volume. That mean we need to use the partial library_size (and molecular_weigth) to calculate each contained library concentration as ng/uL then use the volume ratio to store the concentration. This will likely require a wrapper service for pools concentration calculations.